ghost/i18n/locales/si/portal.json

```
@@ -12,22 +12,22 @@
     "Cancellation reason": "අවලංගු කිරිමට හේතුව",
     "Choose a different plan": "වෙනත් plan එකක් තෝරන්න",
     "Choose your newsletters": "ඔබගේ newsletter වර්ගය තෝරන්න",
-    "Close": "Close",
+    "Close": "වසන්න",
     "Comments": "ප්\u200dරතිචාර",
     "Confirm": "තහවුරු කරන්න",
-    "Continue": "Continue",
+    "Continue": "ඉදිරියට යන්න",
     "Delete account": "ගිණුම Delete කරන්න",
     "Don't have an account?": "ගිණුමක් නොමැතිද?",
     "Email": "Email",
     "Email preference updated.": "Email preference updated.",
     "Email preferences": "Email preferences",
-    "Emails": "Emails",
+    "Emails": "ඊමේල්",
     "Emails disabled": "Emails disabled",
     "Get help": "සහාය ලබාගන්න",
-    "Get notified when someone replies to your comment": "Get notified when someone replies to your comment",
+    "Get notified when someone replies to your comment": "ඔබේ ප්‍රතිචාරයට යම් අයෙකු පිළිතුරු දුන් විට දැනුම් දෙන්න.",
```

No period at the end in existing text
```suggestion
    "Get notified when someone replies to your comment": "ඔබේ ප්‍රතිචාරයට යම් අයෙකු පිළිතුරු දුන් විට දැනුම් දෙන්න",
```